mmedit/models/editors/deepfillv2/two_stage_encoder_decoder.py

```diff
@@ -73,22 +72,11 @@ def forward(self, x):
         return stage1_res, stage2_res
 
     # TODO: study the effects of init functions
-    def init_weights(self, pretrained=None):
-        """Init weights for models.
+    def init_weights(self):
+        """Init weights for models."""
```

Comment:
I think we should add the following code here:
```python
        if self.init_cfg is None and self.init_cfg['type'] == 'Pretrained':
            super().init_weights()
            return
```
Otherwise, this model can never load the pre-trained model.